src/Util/HTTPSignature.php

```diff
@@ -417,7 +417,7 @@ public static function fetch($request, $uid)
 	 * @return \Friendica\Network\HTTPClient\Capability\ICanHandleHttpResponses CurlResult
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function fetchRaw($request, $uid = 0, $opts = ['accept_content' => ['application/activity+json', 'application/ld+json']])
+	public static function fetchRaw($request, $uid = 0, $opts = ['accept_content' => ['application/activity+json, application/ld+json; profile="https://www.w3.org/ns/activitystreams"']])
```

Comment:
This is the correct way of having several values for the same header instead of having a single value by repeated header.

Reply:
I think for a long-term fix, we should allow arrays as an input and format them inside Guzzle as a string.
Because so we could make checks against an array rather than an string

Reply:
There could be arrays that shouldn't be folded but should be added as separate entries.

Reply:
Hm, I think I don't fully understand.

This issue happened because Guzzle sent 2 `Accept` headers:
```
Accept: application/activity+json
Accept: application/ld+json; profile="https://www.w3.org/ns/activitystreams"
```
instead of one comma-separated

**But** this is only an issue for this specific case, because normally we should send separate entries? Or do you mean we are not allowed to send different `Accept` headers, but for other headers we're allowed to?

Reply:
Some headers accept several values by repeating the header, but to my knowledge `Accept:` uses a comma-delimited list for multiple values.